### DISCUSSION TITLE
How to customize tooltip by css?

### DISCUSSION THREAD

**Discussion by kitepad** [c-smile/sciter-js-sdk#156] [Q&A] [answered]
I can customize tooltip text by adding "title" property, but I want to customize tooltip's appearance, for example background-color, border and outline.  Can I do it?

**Comment by MustafaHi** [comment, marked as answer]
```CSS
popup[role="tooltip"]
{
  ...
}
```

https://github.com/c-smile/sciter-sdk/blob/5d2d4b67635a021ff069c417af4c652fa8b9ba5a/samples/tooltips%2B%2B/tooltip-styling.htm
